README.md

```diff
@@ -26,9 +26,13 @@
     </li>
     <li><a href="#react-style-guide">React Style Guide</a></li>
     <li><a href="#commit-style-guide">Commit Style Guide</a></li>
+    <li><a href="#updates">Installation</a></li>
```

Comment:
Why are you adding this to the readme?

Reply:
Sorry, my thinking was to provide a timeline of the changes I had made.

I have removed Updates section from README.md